### PR TITLE
[FastPR] AddGeometries export to Python

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**📝 Description**
This exports the `AddGeometries` of `ModelPart` to Python.

### CODE REVIEW

File: kratos/python/add_model_part_to_python.cpp [hunk 1]
```diff
@@ -949,10 +960,9 @@ void AddModelPartToPython(pybind11::module& m)
         .def("RemoveMasterSlaveConstraintsFromAllLevels", &ModelPart::RemoveMasterSlaveConstraintsFromAllLevels)
         .def("AddMasterSlaveConstraint", ModelPartAddMasterSlaveConstraint)
         .def("AddMasterSlaveConstraints", AddMasterSlaveConstraintsByIds)
-        .def("CreateNewMasterSlaveConstraint",CreateNewMasterSlaveConstraint1, py::return_value_policy::reference_internal)
-        .def("CreateNewMasterSlaveConstraint",CreateNewMasterSlaveConstraint2, py::return_value_policy::reference_internal)
-        .def("__str__", PrintObject<ModelPart>)
-        ;
+        .def("CreateNewMasterSlaveConstraint", CreateNewMasterSlaveConstraint1, py::return_value_policy::reference_internal)
+        .def("CreateNewMasterSlaveConstraint", CreateNewMasterSlaveConstraint2, py::return_value_policy::reference_internal)
+        .def("__str__", PrintObject<ModelPart>);
```
Comment (loumalouomega) [inline kratos/python/add_model_part_to_python.cpp]: I wpould keep ; as its own line

File: kratos/python/add_model_part_to_python.cpp [hunk 2]
```diff
@@ -878,17 +890,17 @@ void AddModelPartToPython(pybind11::module& m)
         .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<bool>)
         .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<int>)
         .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<double>)
-        .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<array_1d<double, 3 > >)
+        .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<array_1d<double, 3>>)
```
Comment (loumalouomega) [inline kratos/python/add_model_part_to_python.cpp:893]: you are mixing clean up and actual changes..., but i will approve

File: kratos/python/add_model_part_to_python.cpp (hunk 1 above)
Comment (loumalouomega) [reply kratos/python/add_model_part_to_python.cpp]:
```suggestion
        .def("__str__", PrintObject<ModelPart>)
        ;
```

File: kratos/python/add_model_part_to_python.cpp (hunk 2 above)
Comment (rubenzorrilla) [reply kratos/python/add_model_part_to_python.cpp:893]: These were automatically added when saving the file. Deactivate the whitespaces.

EDIT: there are much more than whitespaces.... sorry

File: kratos/python/add_model_part_to_python.cpp (hunk 1 above)
Comment (rubenzorrilla) [reply kratos/python/add_model_part_to_python.cpp]: Sorry, blame the autoformatter...

**Review by loumalouomega — APPROVED** [review]

**Comment by rubenzorrilla** [comment]
The Windows CI is failing because one test from the `SystemIdentificationApplication` @sunethwarna 😕

**Pull request merged**